File: 단계별로_풀어보기/2. 조건문/7. 2480.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

sortedArr = list(map(int, input().split()))
sortedArr.sort()

A, B, C = sortedArr

score = 0

if A == B and B == C:
  score = 10000 + 1000 * A
elif A == B or B == C:
  score = 1000 + 100 * B
else:
  score = 100 * C

output(f"{score}")

File: 1. 단계별로_풀어보기/1~9/2. 조건문/7. 2480.js
const fs = require('fs')
const inputs = fs.readFileSync('/dev/stdin').toString()

const [a, b, c] = inputs.split(' ')

const [A, B, C] = [parseInt(a), parseInt(b), parseInt(c)].sort()

let score = 0

if (A === B && B === C) {
  score = 10000 + A * 1000
} // ::
else if (A === B || B === C) {
  score = 1000 + B * 100
} // ::
else {
  score = C * 100
}

process.stdout.write(`${score}`)
